### PR TITLE
Feat/filters

### PR DESCRIPTION
**Pull request by manish591**
#### Added filters functionality

Users can filter out videos based on categories. 

[live preview](https://x0r18d.csb.app/explore)

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Code is well written and clean
UI is functional
PR is good to merge

File: src/components/chips/Chips.css [hunk 1]
```diff
@@ -44,3 +40,12 @@
...
   inset-inline-start: 5px;
 }
+
+.chips__category > label {
+  font-size: 1rem;
+}
+
+.chips__item--selected {
+  background-color: var(--text2);
+  color: var(--light-clr);
```
Comment (tejas-muthya37) [inline src/components/chips/Chips.css:50]: Naming for the CSS variables can be a little more descriptive. text2 seems a little vague.

File: src/utilis/filters.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const getFilterByCategoryItem = (arr, condition) => {
+  if (condition === "All") return arr;
+  return arr.filter((item) => item.category === condition);
```
Comment (tejas-muthya37) [inline src/utilis/filters.js:3]: Good way to handle filters. Kudos. I struggled a lot with this.

**Review by tejas-muthya37 — APPROVED** [review]

**Comment by tejas-muthya37** [comment]
Liked how you handled filters. PR approved. Good luck :)

**Pull request merged**